dali/pipeline/operators/crop/bbox_crop.cc

```diff
@@ -129,20 +133,25 @@ void RandomBBoxCrop<CPUBackend>::RunImpl(SampleWorkspace *ws, const int) {
     labels.emplace_back(*label_data);
   }
 
-  const auto prospective_crop =
-      FindProspectiveCrop(bounding_boxes, labels, SelectMinimumOverlap());
+  while (true) {
```

Comment:
Use some reasonable value for max. number of attempts (100?) if `allow_no_crop` is not specified - otherwise this function may hang.

Reply:
I'm a bit worried about changing this default value, as it severely degrades training accuracy. Additionally, with default threshold `0` and default `allow_no_crop` to `true` the problem should be somewhat alleviated. Maybe I add some info about this to op doc string? To be careful with these parameters and why.

Reply:
while(true)? where is the break condition?